rocksdb_admin/admin_handler.cpp

```diff
@@ -483,6 +483,9 @@ AdminHandler::AdminHandler(
       LOG(INFO) << "Stopping DB deletioin thread ...";
     });
   }
+
+  int upload_MBps = 600; // set as fixed for now.
```

Comment:
yeah, there is a gflag for this

Reply:
I just see s3_download_limit_mb, nothing for upload.

Reply:
yeah, currently, there is limit setup for the write path from EC2 to S3: https://github.com/pinterest/rocksplicator/blob/master/rocksdb_admin/admin_handler.cpp#L1572. 
Should we add 1 more flag?

Reply:
Done, added s3_upload_limit_mb